apps/backend/api/models/util/contextWidgets.js

```diff
@@ -96,7 +96,7 @@ export function replaceHomeWidget ({ widgets, newHomeWidgetId }) {
   // so by here the updatedWidgets array has removed the new home widget from its prior position and its prior peers are settled
 
   // if the old home widget is a chat, we need to move it to the top of the chats widget, otherwise we remove it from the menu
-  if (homeChild.type === 'viewChat') {
+  if (homeChild.type === 'chat' || homeChild.type === 'viewChat') {
```

Comment:
still think we can/should remove 'chat' completely, since it shouldnt be getting used anywhere anymore